server/openapi_server/controllers/tool_controller.py

```diff
@@ -33,4 +35,12 @@ def get_tool_dependencies():  # noqa: E501
 
     :rtype: ToolDependencies
     """
-    return annotators.get_annotators_info(), 200
+    config = Config()
+    tool_dependencies = []
+    for hostname in (
+        config.date_annotator_api_url,
+        config.person_name_annotator_api_url,
+        config.physical_address_annotator_api_url
+    ):
+        tool_dependencies.append(client.get_tool(host=hostname).to_dict())
```

Comment:
This only gets the tool, I realize there's actually a toolDependency endpoint, were you looking for that instead?

Reply:
Hmmm, so the PHI Deidentifier should get present a flattened list of both its dependencies, and its dependencies' dependencies?

I suppose that makes sense if we are allowing annotator docker images to be dependent on other docker images. Is that so @tschaffter ?

Reply:
I'm moving this conversation to issue #94